packages/suite/src/actions/suite/protocolActions.ts

```diff
@@ -62,6 +73,16 @@ export const handleProtocolRequest = (uri: string) => (dispatch: Dispatch, getSt
         if (wcUri) {
             dispatch(walletConnectActions.walletConnectPairThunk({ uri: wcUri }));
         }
+    } else if (uri?.startsWith(SUITE_ANCHOR_DEEPLINK_PREFIX)) {
+        const anchor = uri.replace(SUITE_ANCHOR_DEEPLINK_PREFIX, '');
+
+        if (Object.values(SettingsAnchor).includes(anchor as SettingsAnchorValue)) {
+            const [domain] = anchor.split('/');
+
+            dispatch(
+                goto(mapAnchorToRoute[domain.replace('@', '') as AnchorSettingSection], { anchor }),
+            );
```

Comment:
Two tiny nits:
- replace `@` only at beginning, idk if it's possible at all to have `@` in URL, but I'd rather the  code do _only_ what it's meant to
- extract variable for readability
```suggestion
            const targetRoute = mapAnchorToRoute[domain.replace(/^@/, '') as AnchorSettingSection];
            dispatch(goto(targetRoute, { anchor }));
```

Reply:
Yea, great :+1: